src/manager/evm/manager.rs

```diff
@@ -302,16 +298,14 @@ impl SubnetManager for EthSubnetManager {
         todo!()
     }
 
-    async fn wallet_new(&self, _key_type: WalletKeyType) -> Result<Address> {
-        todo!()
-    }
-
-    async fn wallet_list(&self) -> Result<Vec<Address>> {
-        todo!()
-    }
-
-    async fn wallet_balance(&self, _address: &Address) -> Result<TokenAmount> {
-        todo!()
+    async fn wallet_balance(&self, address: &Address) -> Result<TokenAmount> {
+        let balance = self
+            .ipc_contract_info
+            .provider
+            .clone()
+            .get_balance(payload_to_evm_address(address.payload())?, None)
+            .await?;
+        Ok(TokenAmount::from_atto(balance.as_u128()))
```

Comment:
in the `send_value` PR, we have a `eth_to_fil_amount` method that handles the conversion as str. It should be better than using `u128` as intermediate type.

Reply:
I see you already fixed it there :) Thanks!